roles/setup_efm/tasks/prepare_hba_value_list.yml

```diff
@@ -1,21 +1,46 @@
 ---
-- name: Prepare the list
+- name: Prepare hba non ssl list
   set_fact:
-    temp_list:
-       - contype: "host"
-         users: "{{ pg_efm_user }}"
-         source: "{{ node.private_ip }}/32"
-         databases: "{{ pg_efm_database }}"
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) }} + [
+        {
+          "contype": "host",
+          "users": "{{ pg_efm_user }}",
+          "source": "{{ node.private_ip }}/32",
+          "databases": "{{ pg_efm_database }}"
+          },
+        {
+          "contype": "host",
+          "users": "{{ pg_efm_user }}",
+          "source": "127.0.0.1/32",
+          "databases": "{{ pg_efm_database }}"
+          }
+      ]
   when: not pg_ssl
+  loop: "{{ pg_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
 
-- name: Prepare the list
+- name: Prepare hba non ssl list
```

Comment:
I guess this the "HBA SSL" list, isn't it?

Reply:
fixed